spec/views/posts/index_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index page', type: :feature do
+  before(:each) do
+    @user = User.create! name: 'Doe', email: 'doe@gmail.com', password: '1233456', confirmed_at: Time.now
+    @post = @user.posts.create! title: 'Loren1', text: 'Lorem ipsum amet1 Lorem ipsum amet1'
+    @user.posts.create! title: 'Loren2', text: 'Lorem ipsum amet2 Lorem ipsum amet2'
+    @user.posts.create! title: 'Loren2', text: 'Lorem ipsum amet3'
+    @user.posts.create! title: 'Loren2', text: 'Lorem ipsum amet4'
+    @post.comments.create! text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit', author_id: @user.id
+    @post.comments.create! text: 'Consectetur adipiscing elit', author_id: @user.id
+    Like.create! post_id: @post.id, author_id: @user.id
+    visit user_session_path
+    fill_in 'Email', with: 'doe@gmail.com'
+    fill_in 'Password', with: '1233456'
+    click_on 'Log in'
+    visit user_posts_path(@user)
+  end
+
+  it "I can see the user's profile picture." do
+    expect(page).to have_css('.img', count: 1)
+  end
+
+  it "I can see the user's username." do
+    expect(page).to have_content('Doe')
+  end
+
+  it 'I can see the number of posts the user has written.' do
+    expect(page).to have_content('Number of posts: 4')
+  end
+
+  it "I can see a post's title." do
+    expect(page).to have_content('Loren1')
+    expect(page).to have_content('Loren2')
+  end
```

Comment:
- [ ] As highlighted by the last reviewer. Some of the tests are still failing. see screen-shot below:

![image](https://user-images.githubusercontent.com/51296741/153682150-ae184594-7cfe-4fdb-9ced-a3edc3d9e96b.png)
I will encourage you run `bundle exec rspec` in your terminal if you have not already done so. And take a look at the error message from rspec. It will tell you what line of code is responsible for the failure and why.